main/test/flix/Test.Exp.Force.flix

```diff
@@ -0,0 +1,140 @@
+namespace Test/Exp/Force {
+
+    @test
+    def testUnitForce01(): Bool & Pure =
+        let x = lazy ();
+        () == force x
+
+    @test
+    def testBoolForce01(): Bool & Pure =
+        let x = lazy true;
+        true == force x
```

Comment:
Add the false case too. For Booleans it is easy to slip up, so I always try to have both cases covered.